test/setup-ca.sh

```diff
@@ -13,23 +13,24 @@ if [ $cfssl_found -eq 0 ]; then
     echo "cfssl tools not found, Please install cfssl and cfssljson."
     exit 1
 fi
-
+set -x
```

Comment:
This probably needs to be removed, right?

Reply:
oops. sorry. I will fix it.

Reply:
Done.